chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -850,45 +849,45 @@ public static DataSet mathFunction(final DataSet function1, final DataSet functi
             switch (op) {
             case ADD:
                 newY = Y1 + Y2;
-                nEYN = Math.hypot(eyn1, eyn2);
-                nEYP = Math.hypot(eyp1, eyp2);
+                nEYN = java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = java.lang.Math.hypot(eyp1, eyp2);
                 break;
             case SUBTRACT:
                 newY = Y1 - Y2;
-                nEYN = Math.hypot(eyn1, eyn2);
-                nEYP = Math.hypot(eyp1, eyp2);
+                nEYN = java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = java.lang.Math.hypot(eyp1, eyp2);
                 break;
             case MULTIPLY:
                 newY = Y1 * Y2;
-                nEYN = Math.hypot(Y2 * eyn1, Y1 * eyn2);
-                nEYP = Math.hypot(Y2 * eyp1, Y1 * eyp2);
+                nEYN = java.lang.Math.hypot(Y2 * eyn1, Y1 * eyn2);
+                nEYP = java.lang.Math.hypot(Y2 * eyp1, Y1 * eyp2);
                 break;
             case DIVIDE:
                 newY = Y1 / Y2;
-                nEYN = Math.hypot(eyn1 / Y2, newY * eyn2 / Y2);
-                nEYP = Math.hypot(eyp1 / Y2, newY * eyp2 / Y2);
+                nEYN = java.lang.Math.hypot(eyn1 / Y2, newY * eyn2 / Y2);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.hypot' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)